src/app/server-cluster/StandardServerCluster.h

```diff
@@ -0,0 +1,88 @@
+/*
+ *    Copyright (c) 2025 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+#pragma once
+
+#include <app/server-cluster/ServerClusterInterface.h>
+
+namespace chip {
+namespace app {
+
+/// Provides an implementation of most methods for a `ServerClusterInterface`
+/// to make it easier to implement spec-compliant classes.
+///
+/// In particular it does:
+///   - maintains a data version and provides `IncreaseDataVersion`. Ensures this
+///     version is spec-compliant initialized (with a random value)
+///   - Provides default implementations for most virtual methods EXCEPT:
+///       - ReadAttribute (since that one needs to handle featuremap and revision)
+///       - GetClusterId (since every implementation is for different clusters)
+///
+///
+class StandardServerCluster : public ServerClusterInterface
```

Comment:
Is this really "Standard"?  Or is it more of a "DefaultServerCluster"?

Reply:
I had just picked a filler word to have a subclass. Default sounds good ... it is a `with default implementations` bit.